src/apify_error.js

```diff
@@ -16,4 +18,18 @@ export default class ApifyClientError extends Error {
 
         Error.captureStackTrace(this, ApifyClientError);
     }
+
+    toString() {
+        let details = '';
+
+        if (this.details) {
+            const items = [];
+            _.mapObject(this.details, (val, key) => {
```

Comment:
I guess I'd prefer something like:

```js
details = Object.entries(this.details)
    .map(([key, val]) => `${key}=${JSON.stringify(val)}`)
    .join(', ');
```

but that's probably just me not being very used to underscore.